### DISCUSSION TITLE
本人于9月14日台风天返校补课 算今天以连续上学8天 发此公告 望周知

### DISCUSSION THREAD

**Discussion by KELERSAMA** [share121/inter-knot#4871] [灌水]
![image](https://github.com/user-attachments/assets/052e6cd5-dd2b-4046-966a-1a74a12ea356)

https://github.com/user-attachments/assets/c57dbff5-35dd-4bf3-9dc9-375e56b5d1d0

本人在倡导幸福教育的公立半私立学校当畜生 因为中秋假期 于是要求被于9月14日台风天返校补课 于今日连同年级2个班前往校长办公室起义遭打压 算今天以连续上学8天 已经生无可恋 算调休比周末假期还少1天 仅半天 发此公告 望周知

**Comment by FlyMan2004** [comment]
已阅，下一个。
